task/01-strings-tasks.js

```diff
@@ -65,8 +63,8 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, John Doe!' => 'John Doe'
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
-function  extractNameFromTemplate(value) {
-  throw new Error('Not implemented');
+function  extractNameFromTemplate(value) { 
+  return  value.slice(value.indexOf(' ') + 1, value.length - 1);  
```

Comment:
the solution can be shorten, try to avoid `indexOf` because it's equal to array iteration